cli/src/test/resources/ide-projects/jasypt/repository/java/java/default/bin/java.cmd

```diff
@@ -0,0 +1,3 @@
+@echo off
```

Comment:
if you do not want to test OS specific behavior you can remove the `java.cmd` so you only have to maintain `java` bash script.